docs/install/spark_cluster_mode.md

```diff
@@ -202,3 +202,65 @@ After running a single paragraph with Spark interpreter in Zeppelin, browse `htt
 
 <img src="../assets/themes/zeppelin/img/docs-img/mesos_frameworks.png" />
 
+
+## Zeppelin on CDH
+Cloudera officially provide docker container so we can easily build CDH docker environment following the [link](https://www.cloudera.com/documentation/enterprise/5-6-x/topics/quickstart_docker_container.html).
```

Comment:
link to https://www.cloudera.com/documentation/enterprise/latest.html
the latest CDH release is 5.8